src/Audit/Drupal/SolrVersion.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drutiny\Audit\Drupal;
+
+use Drutiny\Audit;
+use Drutiny\Sandbox\Sandbox;
+use Drutiny\Annotation\Param;
+
+/**
+ * Check the version of Drupal project in a site.
+ * @Param(
+ *  name = "module",
+ *  description = "The module to version information for",
+ *  type = "string"
+ * )
+ * @Param(
+ *  name = "version",
+ *  description = "The static version to check against.",
+ *  type = "string"
+ * )
+ * )
+ */
+class SolrVersion extends Audit
+{
+    /**
+     * @inheritdoc
+     */
+    public function configure()
+    {
+        $this->addParameter(
+            'module',
+            static::PARAMETER_OPTIONAL,
+            'Module to check.'
+        );
+        $this->addParameter(
+            'version',
+            static::PARAMETER_OPTIONAL,
+            'Module to check.'
+        );
+    }
+
+    public function audit(Sandbox $sandbox)
+    {
+        $module = $sandbox->getParameter('module');
+        // Version of SOLR we need to find if exists.
+        $version_searched = $sandbox->getParameter('version');
+
+
+        $info = $sandbox->drush(['format' => 'json'])->pmList();
```

Comment:
$sandbox is deprecated. Use this instead:

```php
     $info = $this->target->getService('drush')
          ->pmList([
            'format' => 'json',
            'type' => 'module'
          ])
          ->run(function ($output) {
          return json_decode($output, true);
        });
```

see: https://github.com/drutiny/drutiny/blob/3.0.x/src/Audit/Drupal/ModuleAnalysis.php